agent/containers/images/visualizers/live-metric-visualizer/prom_ds.py

```diff
@@ -0,0 +1,92 @@
+import json
+import requests
+import subprocess
+import time
+
+graf_base = "http://localhost:3000/"
```

Comment:
I'd like to see a description of why you created this and what it does so everyone doesn't necessarily have to read the entire program to find out.

The name `prom_ds.py` also doesn't help much; a name more like `configure_grafana.py` would at least give a potential reader an idea of what they're going to see.